### PR TITLE
Adding usecases and requirements to Rucio

### PR DESCRIPTION
**Pull request by Thysk**
# Summary
This PR brings some requirements that users should consider when choosing Rucio as their data management solution
as well as a couple of usecases that users may use Rucio for.

### CODE REVIEW

**Comment by Thysk** [comment]
the check says there is a dead link, but when following the link I get to the page
can someone else check the 'dead' link please?

File: content/en/users/data/management/rucio/_index.md [hunk 1]
```diff
@@ -26,6 +26,48 @@ Petabytes. The fact that the underlying Rucio infrastructure is managed by STFC,
 allows communities to easily start using and/or test Rucio with little setup
 cost.
 
+## Requirements to consider to use Rucio
+
+For Rucio to mangage your data in this setup, Rucio will need X509 certificate
```
Comment (andrea-manzi) [inline content/en/users/data/management/rucio/_index.md:31]:
```suggestion
For Rucio to manage your data in this setup, Rucio will need X509 certificate
```

File: content/en/users/data/management/rucio/_index.md [hunk 2]
```diff
@@ -26,6 +26,48 @@ Petabytes. The fact that the underlying Rucio infrastructure is managed by STFC,
...
+access, or soon, through
+[EGI Check-in](../../../../providers/check-in/_index.md) to:
+
+- Your experiments Storage Element that is X509 capable
+- Your experiments Tape Archive that is X509 capable
+- Your experiments Voms server information to check Users credentials against
+
+Rucio is a system that sits on top of already established storage elements to
+unify users access for data management, and retrival. Rucio consists of a
```
Comment (andrea-manzi) [inline content/en/users/data/management/rucio/_index.md:40]:
```suggestion
unify users access for data management, and retrieval. Rucio consists of a
```

File: content/en/users/data/management/rucio/_index.md [hunk 3]
```diff
@@ -26,6 +26,48 @@ Petabytes. The fact that the underlying Rucio infrastructure is managed by STFC,
...
+database of the storage element's details, users and their access credential
+information, and access levels, and the data and its location, not a direct data
```
Comment (andrea-manzi) [inline content/en/users/data/management/rucio/_index.md:42]:
```suggestion
information, access levels and the data and its location. Rucio is  not a direct data
```

File: content/en/users/data/management/rucio/_index.md [hunk 4]
```diff
@@ -26,6 +26,48 @@ Petabytes. The fact that the underlying Rucio infrastructure is managed by STFC,
...
+
+## Rucio Use Cases
+
+Rucio is a data management software, that integrates with your experiments
+currently provisioned storage. Therefore, this section will deal with usecases
+that you should consider Rucio for.
+
+### Management of data for archive when no longer used actively
+
+One that may be useful is if your experiment has data that is newly generated,
```
Comment (andrea-manzi) [inline content/en/users/data/management/rucio/_index.md:53]: not so much clear this paragraph

File: content/en/users/data/management/rucio/_index.md [hunk 5]
```diff
@@ -26,6 +26,48 @@ Petabytes. The fact that the underlying Rucio infrastructure is managed by STFC,
...
+precious and needs to be preserved. To ensure the integrity of the data, Rucio
+can be instructed by two rules, one to write the data to a cheap long term
+archive of tape storage, the second to ensure the data is also available on HDDs
+or, if available SSDs. Then as the usage of this data declines, the data on the
+more accessible HDD or SSD can be removed, in the knowledge that if the data is
+ever required again, Rucio can stage the data from tape back to disk.
+
+### Management of data between sites for jobs
+
+Another useful usecase for Rucio is to mangage the data between different sites
```
Comment (andrea-manzi) [inline content/en/users/data/management/rucio/_index.md:64]:
```suggestion
Another useful use case for Rucio is to manage the data between different sites
```

File: content/en/users/data/management/rucio/_index.md (hunk 3 above)
Comment (Thysk) [reply content/en/users/data/management/rucio/_index.md]: changes made

File: content/en/users/data/management/rucio/_index.md (hunk 5 above)
Comment (Thysk) [reply content/en/users/data/management/rucio/_index.md]: changes made

File: content/en/users/data/management/rucio/_index.md (hunk 4 above)
Comment (Thysk) [reply content/en/users/data/management/rucio/_index.md]: rewrote in latest PR, hopefully makes more sense to you

File: content/en/users/data/management/rucio/_index.md (hunk 2 above)
Comment (Thysk) [reply content/en/users/data/management/rucio/_index.md]: changes made

File: content/en/users/data/management/rucio/_index.md (hunk 1 above)
Comment (Thysk) [reply content/en/users/data/management/rucio/_index.md]: changes made

**Review by andrea-manzi — APPROVED** [review]
LGTM!

**Pull request merged**